src/components/ScrollableContainer.tsx

```diff
@@ -38,12 +36,22 @@ const ScrollableContainer: FC<Props> = ({
     childContainer.setAttribute("style", style);
   };
 
+  useEffect(() => {
```

Comment:
let's revert all the changes in `ScrollableContainer` as the component is used in many other places. I would put this logic in `ProfileSelector` and the `useEffect` can track the `selected` prop.

As a side note, an `useEffect` hook with no dependencies array is an anti-pattern in React and should be avoided.